public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -265,6 +266,16 @@ export function LogsTableWrap(props: Props) {
     // The panel state is updated when the user interacts with the multi-select sidebar
   }, [currentDataFrame, getColumnsFromProps]);
 
+  const [sidebarWidth, setSidebarWidth] = useState(220);
+  const totalWidth = props.width;
+  const [tableWidth, setTableWidth] = useState(totalWidth - sidebarWidth);
+
+  useEffect(() => {
+    setTableWidth(totalWidth - sidebarWidth);
+    // We don't want to update the tableWidth when the sidebar width changes, only when the window width changes.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [totalWidth]);
+
```

Comment:
Do we need a state for `tableWidth`? It seems that it is always `props.width - sidebarWidth`, so we could just use that, right?

Reply:
Nope! Fixed 🙇